guides/serial.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: guide
+banner: images/coolterm-banner.png
+---
+
+# Print debugging over USB
+
+## Windows
+
+Download the [serial driver](http://developer.mbed.org/media/downloads/drivers/mbedWinSerial_16466.exe).
+
+Plug the mbed into your computer, and then run the installer.
+
+Download [termite](http://www.compuphase.com/software/termite-3.1.exe) which will be used for displaying the print output from the mbed.
+
+## OS X
+
```

Comment:
On the windows version it says you need the mbed plugged in. It might seem obvious, but might not be